syntax="proto2";
package peleus.modules.configure;

message server_conf {
    required string main_server_conf_path = 1;
    required int32 idle_timeout_sec = 2;
    required int32 max_concurrency  = 3;
    required int32 internal_port = 4;
    required int32 port = 5;
    required int32 num_threads = 6;
}
